packages/vaadin-combo-box/src/vaadin-combo-box-light.d.ts

```diff
@@ -86,3 +86,13 @@ declare global {
 }
 
 export { ComboBoxLight };
+
+/**
+ * @deprecated Import `ComboBoxLight` from `@vaadin/vaadin-combo-box` instead.
+ */
+export type ComboBoxLightElement = ComboBoxLight;
```

Comment:
Combobox is a different scenario, where the package has not been moved yet, but still has renamed the element.

I decided to add the legacy alias to the actual components code for now. We should separate this, when we move the package.